framework/src/modules/http_api/controllers/peers.js

```diff
@@ -60,33 +63,34 @@ PeersController.getPeers = async function(context, next) {
 	// Remove filters with null values
 	filters = _.pickBy(filters, v => !(v === undefined || v === null));
 
-	await channel.invoke('chain:getPeers', [
-		filters,
-		async (err, data) => {
-			if (err) {
-				return next(err);
-			}
+	try {
+		const data = await channel.invoke('chain:getPeers', [filters]);
 
-			const clonedData = _.cloneDeep(data);
-			const filteredData = clonedData.map(peer => {
-				const { updated, ...filtered } = peer;
-				return filtered;
-			});
+		const clonedData = _.cloneDeep(data);
+		const filteredData = clonedData.map(peer => {
+			const { updated, ...filtered } = peer;
+			return filtered;
+		});
 
-			const peersCount = await channel.invoke('chain:getPeersCountByFilter', [
-				_.cloneDeep(filters),
-			]);
+		const peersCount = await channel.invoke('chain:getPeersCountByFilter', [
+			_.cloneDeep(filters),
+		]);
 
-			return next(null, {
-				data: filteredData,
-				meta: {
-					offset: filters.offset,
-					limit: filters.limit,
-					count: peersCount,
-				},
-			});
-		},
-	]);
+		return next(null, {
+			data: filteredData,
+			meta: {
+				offset: filters.offset,
+				limit: filters.limit,
+				count: peersCount,
+			},
+		});
+	} catch (err) {
+		error = new ApiError(err, apiCodes.PROCESSING_ERROR);
```

Comment:
Why is this a `PROCESSING_ERROR`?

Reply:
Reset back to: next(err) as this is breaking change, thanks!